t/tac.t

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use Test::More tests => 3;
+
+sub _lines2re
+{
+    return join( qq#\r?\n#, @_ ) . qq#\r?\n?#;
+}
+
+sub test_tac
+{
+    local $Test::Builder::Level = $Test::Builder::Level + 1;
+    my ($args) = @_;
+
+    my @files = @{ $args->{files} };
+    my @flags = @{ $args->{flags} };
+
+    # die "@flags @files";
+    my $re = _lines2re( @{ $args->{lines} } );
+    return like( scalar(`$^X -Ilib bin/tac @flags @files`),
+        qr#\A$re\z#ms, $args->{blurb} );
+}
+
+# TEST
+test_tac(
+    {
+        blurb => "multiple files tac",
+        files => [
+            qw( t/data/sort/three-words.txt t/data/sort/ints1.txt t/data/sort/letters1.txt )
+        ],
+        flags => [qw/ /],
+        lines => [ split /\n/, <<'EOF'],
+column by pencil
+row by row
+mooing yodelling dog
+mooing persistent cat
+a little love
+the meta protocol
+based little mint
+the wonderful unicorn
+15
+100
+48
+33
+12
+35
+44
+17
+59
+92
+53
+29
+46
+2
+70
+64
+54
+13
+85
+23
+82
+57
+38
+32
+56
+99
+34
+83
+19
+77
+9
+79
+60
+4
+89
+86
+1
+52
+45
+78
+95
+11
+90
+40
+98
+81
+43
+93
+91
+8
+21
+22
+39
+69
+96
+24
+68
+67
+31
+87
+72
+16
+5
+76
+62
+71
+6
+42
+97
+27
+49
+50
+94
+61
+88
+30
+14
+65
+3
+73
+74
+26
+58
+80
+47
+37
+41
+36
+75
+63
+7
+51
+28
+25
+84
+10
+18
+55
+20
+66
+e
+f
+c
+a
+d
+b
+EOF
+    }
+);
+
+# TEST
+test_tac(
+    {
+        blurb => "-s flag",
+        files => [qw( t/data/tac/a-sep.txt )],
+        flags => [qw/ -s a /],
+        lines => [ split /\n/, <<'EOF'],
+threeatwoaonea
+EOF
+    }
+);
+
+# TEST
+test_tac(
+    {
+        blurb => "-s and -r flags",
+        files => [qw( t/data/tac/a-sep.txt )],
+        flags => [qw/ -rs "[az]" /],
+        lines => [ split /\n/, <<'EOF'],
+threeatwoaonea
+EOF
+    }
+);
+
+__END__
+
+=head1 COPYRIGHT & LICENSE
```

Comment:
You need to license everything in the same way that the entire package is licenses. In this case, it's exactly the Artistic License 2.0 (as seen in LICENSE). It can't be a future license that does not exist.

Reply:
hi @briandfoy ! I hope you don't take it the wrong way but you seem a little too pedantic and stubborn and bikeshedding here - see http://shlomifishswiki.branchable.com/Never_Try_to_Please_Everyone/ . While [IANAL](https://en.wikipedia.org/wiki/IANAL) but it is not inconceivable that one day there will be The Artistic License 2.1 or 3.0 or whatever. You can be a bit more tolerant. [Hakuna matata](https://duckduckgo.com/?q=hakuna+matata&atb=v140-1&ia=about) .